### ISSUE TITLE
FIX: Zeek Intel not working

### ISSUE THREAD

**Issue by dougburks** [Security-Onion-Solutions/securityonion#3850]
From https://github.com/Security-Onion-Solutions/securityonion/discussions/3830:

> I am unable to get Zeek Intel working. I checked the reporter log and it is blank. I looked to see if the intel.log file was being generated and that is also missing. I added 8.8.8.8 and ran a test ping to it and zeek is not generating any hits. Please help point me in the correct direction. I saw another post mentioning in relation to this, but not directly this. is Zeek Intel working properly? Thanks.

![seconion2-intel-dat](https://user-images.githubusercontent.com/12386911/114306660-7308b380-9aaa-11eb-8b61-6cc68ebad4e7.PNG)

_Originally posted by @bdorr1105 in https://github.com/Security-Onion-Solutions/securityonion/discussions/3830_

**Comment by dougburks** [comment]
I confirmed that one of my PROD deployments had Zeek Intel working properly until it was upgraded to 2.3.40. After the upgrade when I try to trigger an Intel hit, nothing gets logged in intel.log but I see the following in reporter.log:
```
{"ts":"2021-04-13T11:37:47.442195Z","level":"Reporter::WARNING","message":"failed to convert remote event 'Intel::match_remote' arg #0, got vector, expected record","location":""}
{"ts":"2021-04-13T11:37:47.455320Z","level":"Reporter::WARNING","message":"failed to convert remote event 'Intel::match_remote' arg #0, got vector, expected record","location":""}
```

I then tested a 2.3.40 Eval installation and Zeek Intel seems to work there, so it's possible that the difference is Eval installations run Zeek in standalone mode instead of cluster mode.

**Comment by bdorr1105** [comment]
I went and did SSH to my standalone to look in /nsm/zeek/logs/current and do not even see a reporter.log / Also, where is the intel.dat supposed to exist. I have 2 locations with the same because of what I was reading in the documentation.

/opt/so/saltstack/local/salt/zeek/policy/intel/intel.dat

That is what is in the picture above, I have nothing being reported. I went back to the dat file and removed all non single space tabs and have not seemed to make any progress

**Comment by dougburks** [comment]
We've been chatting with the Zeek devs in the Zeek slack today and this is looking like a bug in Zeek 4.

**Comment by dougburks** [comment]
This bug has been confirmed in Zeek 4.0.0 and is scheduled to be fixed in Zeek 4.0.1:
https://github.com/zeek/zeek/issues/1506

**Comment by TOoSmOotH** [comment]
Zeek has been updated to 4.0.1. Will close when nightly tests pass.
